clients/client-python/gravitino/dto/requests/model_version_link_request.py

```diff
@@ -59,7 +59,7 @@ def validate(self):
 
         for alias in self._aliases or []:
             if not self._is_not_blank(alias):
-                raise IllegalArgumentException('Alias must not be null or empty')
```

Comment:
Do we need this change?

Reply:
This is a style fix automatically changed by gradle python build, it is no harm to include in this PR.